spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -1582,19 +1582,29 @@ def paste(self):
         text has 'CRLF' EOL chars
         """
         clipboard = QApplication.clipboard()
-        # This is here to prevent pasting mime data in the Editor.
-        # See issue 8566 for the details.
-        if clipboard.mimeData().hasUrls():
-            return
-        else:
-            text = to_text_string(clipboard.text())
-            if len(text.splitlines()) > 1:
-                eol_chars = self.get_line_separator()
-                text = eol_chars.join((text + eol_chars).splitlines())
-                clipboard.setText(text)
-            # Standard paste
-            TextEditBaseWidget.paste(self)
-            self.document_did_change(text)
+        text = to_text_string(clipboard.text())
+        # This is here to make copied files/folders to be pasted as text path.

```

Comment:
I'll put on my @CAM-Gerlach hat here but this line would be more concise as.

```python
# Have copied file and folder urls pasted as text paths.`
# See issue: 8566 and PR: 8576 for details.
```

Reply:
Heh :) I was itching to point that out but I didn't want to be too @CAM-Gerlach -y

Reply:
> I was itching to point that out 

I thought you might be ;)